AV/VehicleAutomationLevels.rdf

```diff
@@ -175,18 +177,20 @@
 						<owl:unionOf rdf:parseType="Collection">
 							<rdf:Description rdf:about="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/AutomatedDrivingSystem">
 							</rdf:Description>
-							<rdf:Description rdf:about="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/ConventionalDriver">
+							<rdf:Description rdf:about="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/InVehicleDriver">
 							</rdf:Description>
 						</owl:unionOf>
 					</owl:Class>
 				</owl:someValuesFrom>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">ADS dual-mode vehicle</rdfs:label>
-		<skos:definition xml:lang="en">ADS-equipped vehicle designed for both driverless operation and operation by a conventional driver for complete trips</skos:definition>
-		<AnnotationVocabulary:adaptedFrom>SAE-J3016 (2018), sec. 3.12</AnnotationVocabulary:adaptedFrom>
-		<AnnotationVocabulary:explanatoryNote xml:lang="en">An ADS feature that is usable during only part of a trip, such as a feature designed to perform the complete DDT during traffic jams on freeways, would not be sufficient to classify its host vehicle as a dual-mode vehicle because it would not be capable of driverless operation for a complete trip.</AnnotationVocabulary:explanatoryNote>
-		<AnnotationVocabulary:explanatoryNote xml:lang="en">An ADS-equipped dual-mode vehicle may be designed such that it can be dispatched in driverless operation.</AnnotationVocabulary:explanatoryNote>
+		<skos:definition xml:lang="en">ADS-equipped vehicle designed to enable either driverless operation under routine/normal operating conditions within its given ODD (if any), or operation by an in-vehicle driver, for complete trips</skos:definition>
+		<AnnotationVocabulary:adaptedFrom>SAE-J3016 APR2021, sec. 3.32.2</AnnotationVocabulary:adaptedFrom>
+		<AnnotationVocabulary:adaptedFrom>SAE-J3016 JUN2018, sec. 3.12</AnnotationVocabulary:adaptedFrom>
```

Comment:
Since the JUN2018 version does not include InVehicleDriver, this reference is wrong. This applies to all similar cases in this pull-request.